WordPress/Classes/ViewRelated/Likes/LikesListController.swift

```diff
@@ -370,20 +365,18 @@ private extension LikesListController {
     }
 
     func setupHeaderCell(cell: NoteBlockHeaderTableViewCell, group: FormattableContentGroup) {
-        cell.attributedHeaderTitle = nil
-        cell.attributedHeaderDetails = nil
-
         guard let gravatarBlock: NotificationTextContent = group.blockOfKind(.image),
             let snippetBlock: NotificationTextContent = group.blockOfKind(.text) else {
                 return
         }
 
-        cell.attributedHeaderTitle = formatter.render(content: gravatarBlock, with: HeaderContentStyles())
-        cell.attributedHeaderDetails = formatter.render(content: snippetBlock, with: HeaderDetailsContentStyles())
-
         // Download the Gravatar
         let mediaURL = gravatarBlock.media.first?.mediaURL
-        cell.downloadAuthorAvatar(with: mediaURL)
+        cell.configure(
+            text: snippetBlock.text ?? "",
+            avatarURL: notification?.kind == .commentLike || notification?.kind == .follow ? mediaURL : nil,
+            action: { [weak self] in self?.delegate?.didSelectHeader!() }
+        )
     }
```

Comment:
[Suggestion] If the `mediaURL` is `nil` (like when a user doesn't have an avatar), then the content preview image won’t appear, even if `notification.kind` is `commentLike` or `follow`. I recommend using the `Avatar` convenience initializer when instantiating `ContentPreview.ImageConfiguration`:

```swift
    init(avatar: Avatar) {
        self.init(url: Self.avatarURL(from: avatar), placeholder: Image("gravatar"))
    }

    struct Avatar {
        let url: URL?
        let email: String?
        let size: CGFloat

        init(url: URL?, email: String?, size: CGFloat = 80) {
            self.url = url
            self.email = email
            self.size = size
        }
    }
```

[Here is en example.](https://github.com/wordpress-mobile/WordPress-iOS/blob/feature/notifications_refresh_p2/WordPress/Classes/ViewRelated/Comments/CommentDetailViewController.swift#L440)